features/correction/generator.py

```
@@ -164,17 +206,58 @@ def _validate(
         return errors
 
 
-def _load_max_retries() -> int:
+def _load_correction_config() -> CorrectionConfig:
+    config_path = Path(__file__).resolve().parent / "config" / "correction.yaml"
+
+    try:
+        with config_path.open(encoding="utf-8") as file:
+            config = yaml.safe_load(file)
+    except OSError as exc:
+        raise ValueError(f"첨삭 설정 파일을 읽을 수 없습니다: {exc}") from exc
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)